src/ankita/assignment/string/SumOfDoubleNumbersFromString.java

```diff
@@ -0,0 +1,37 @@
+/*Assignment : 43 [23rd Dec'2020]
+
+WAP to sum numbers from below String
+
+input: I have 5.6 years of exp in automation and 2.3 in manual testing
+output: 7.9
+
+hint: Double's parse method 
+use exception handling
+*/
+package ankita.assignment.string;
+
+public class SumOfDoubleNumbersFromString {
+
+	void getSumOfDoubleNumbersFromString(String str) {
+		System.out.println("Input :" + str);
+		String[] arr = str.split(" ");
+		double sum = 0;
+		for (int i = 0; i < arr.length; i++) {
+			String word = arr[i];
+			try {
+				if (Character.isDigit(arr[i].charAt(0))) {
+					sum = sum + Double.parseDouble(word);
+				}
+			} catch (NumberFormatException ne) {
+				System.out.println("Number Format Exception");
+			}
+		}
+		System.out.println("Output:" + sum);
```

Comment:
this will print ans as 7.899990 right, but we want 2 digit only after decimal, so for that you can use below function
here you have two methods to handle this scenario,

1. System.out.printf("%.1f", sum);
2. String.format("%.1f",sum);